hubmap/frontend/templates/base.html

```diff
@@ -0,0 +1,23 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>HuBMAP</title>
+     {% csrf_token %}
+</head>
+<nav>
+    <ul>
+    <li><a href="{% url 'listall' %}">Browse All Entities</a></li>
+    <li><a href="{% url 'entities' 'sample' %}">Browse all Samples</a> </li>
+    <li><a href="{% url 'entities' 'donor' %}">Browse all Donors</a> </li>
+    <li><a href="{% url 'entities' 'dataset' %}">Browse all Datasets</a> </li>
+    <li><a href="{% url 'help' %}">Help</a> </li>
+</ul>
+</nav>
+<body>
+<h3>{{ title }} </h3>
```

Comment:
Either a padding space on both sides, or none. Be consistent.

Reply:
fixed